app/src/main/java/org/thoughtcrime/securesms/ApplicationContext.java

```diff
@@ -501,7 +501,8 @@ private void loadEmojiSearchIndexIfNeeded() {
         });
     }
 
-    public void clearAllData(boolean isMigratingToV2KeyPair) {
+    // Method to clear the local data - returns true on success otherwise false
```

Comment:
Do you mean like this?
```
/**
 * Clear all local profile data and message history then restart the app after a brief delay.
 * @param isMigratingToV2KeyPair whether we're upgrading to a more recent V2 key pair or not.
 * @return true on success, false otherwise.
 */
```